ext/ffi/lib.rs

```diff
@@ -1194,76 +1194,89 @@ fn op_ffi_get_static<'scope>(
 ) -> Result<serde_v8::Value<'scope>, AnyError> {
   let resource = state.resource_table.get::<DynamicLibraryResource>(rid)?;
 
-  let data_ptr = resource.get_static(name)? as *const u8;
+  let data_ptr = resource.get_static(name)?;
 
   Ok(match static_type {
     NativeType::Void => {
       return Err(type_error("Invalid FFI static type 'void'"));
     }
     NativeType::U8 => {
       let result = unsafe { ptr::read_unaligned(data_ptr as *const u8) };
-      let number = v8::Number::new(scope, result as f64);
-      serde_v8::from_v8(scope, number.into())?
+      let number: v8::Local<v8::Value> =
+        v8::Number::new(scope, result as f64).into();
```

Comment:
Why not use v8::Integer::new() ?

Converting to float and then back to int has a lot of overhead.

Reply:
Ah, this is a copy-paste error indeed. I think this was what I was originally supposed to fix in this PR (I had a note to self to give statics some love) but forgot all about it.

Reply:
Massive thanks for noticing this! Leaving this in would've been the height of idiocy and I was completely blind to it.